source/sass/mofo-bootstrap/overrides/_variables.scss

```diff
@@ -23,7 +24,6 @@ $btn-border-radius: 0;
 $btn-border-radius-sm: 0;
 $btn-border-radius-lg: 0;
 $btn-font-weight: 700;
-$border-shadow-color: transparentize($black, 0.85);
```

Comment:
This isn't a Bootstrap variable so I've moved to [source/sass/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-d2049013185550a0f0a347ec91292d0f766e9bdd8dbdd9a94769d5fa8673723d)